.github/workflows/ci-test-suite.yml

```
@@ -5,23 +5,43 @@ on:
-    if: ${{ !contains(github.event.pull_request.title, '[WIP]') && !contains(github.event.pull_request.title, '[wip]') && !contains(github.event.pull_request.title, '[Wip]') && !contains(github.event.pull_request.title, '[DRAFT]') && !contains(github.event.pull_request.title, '[draft]') && !contains(github.event.pull_request.title, '[Draft]') }}
+    outputs:
+      packages: ${{ steps.filter.outputs.changes }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
 
-    defaults:
-      run:
-        working-directory: ./packages/react
+      - name: Detect file changes
+        uses: dorny/paths-filter@v2
```

actionlint (1.7.4)

the runner of "dorny/paths-filter@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)